enterprise/server/raft/replica/replica.go

```diff
@@ -743,194 +669,71 @@ func (sm *Replica) deleteStoredFiles(start, end []byte) error {
 	return nil
 }
 
-func (leftReplica *Replica) copyDataToReplica(rightReplica *Replica, insertBatch *pebble.Batch, start, end []byte) error {
-	ctx := context.TODO()
-	iter := leftReplica.db.NewIter(&pebble.IterOptions{
-		LowerBound: keys.Key(start),
-		UpperBound: keys.Key(end),
+func (sm *Replica) copyStoredFiles(req *rfpb.CopyStoredFilesRequest) (*rfpb.CopyStoredFilesResponse, error) {
+	appliesToThisReplica := sm.splitAppliesToThisReplica(req.GetSourceRange())
+	if !appliesToThisReplica {
+		sm.log.Debugf("Got copyStoredFiles cmd but it doesn't apply to this replica; skipping")
+		return &rfpb.CopyStoredFilesResponse{}, nil
+	}
+
+	localReplica, err := sm.store.GetReplica(req.GetTargetRangeId())
```

Comment:
can we call this targetReplica or destReplica? both of the replicas involved are local

Reply:
Done